rules/windows/process_creation/win_susp_findstr.yml

```diff
@@ -0,0 +1,32 @@
+title: Abusing Findstr for Defense Evasion 
+id: bf6c39fc-e203-45b9-9538-05397c1b4f3f
+description: Attackers can use findstr to hide their artifacts or search specific strings and evade defense mechanism
+author: 'Furkan CALISKAN, @caliskanfurkan_, OSCD Community'
```

Comment:
could you please also put "oscd.community" here?

Reply:
Done